docs/standard/assembly/reference-assemblies.md

```diff
@@ -0,0 +1,74 @@
+
+## Using reference assemblies
+
+To use certain APIs from your project, you must add references to their assemblies. You can add references either to implementation assemblies directly or to reference assemblies. We strongly recommend that you use reference assemblies whenever they are available, because doing so ensures that you are using only API members that are supported in the target version and are meant to be used by API designers (in other words, not taking a dependency on implementation details).
+
+Reference assemblies for the .NET Framework libraries are distributed with targeting packs. You can obtain them by downloading a standalone installer or by selecting a component in Visual Studio installer. For more information, see [Install the .NET Framework for developers](../../framework/install/guide-for-developers.md). Reference assemblies for .NET Core libraries are managed as NuGet packages. For example, the [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) package contains reference assemblies that represent common .NET Core APIs. To import these packages, you must first install the SDK for the target platform version you plan to use. For more information, see [Packages, metapackages and frameworks](../../core/packages.md) in the .NET Core Guide.
+
+When you add references to .NET Framework assemblies in Visual Studio using the **Add reference** dialog, you select an assembly from the list, and Visual Studio automatically finds reference assemblies that correspond to the target framework version selected in your project. When you add references to these assemblies in the command line by using the `-reference` compiler option ([in C#](../../csharp/language-reference/compiler-options/reference-compiler-option.md) and in [Visual Basic](../../visual-basic/reference/command-line-compiler/reference.md)) or by using the <xref:Microsoft.CodeAnalysis.Compilation.AddReferences%2A?displayProperty=nameWithType> method in the Roslyn API, you must manually specify reference assembly files for the correct target platform version. .NET Framework reference assembly files are located in the *%ProgramFiles(x86)%\\Reference Assemblies\\Microsoft\\Framework\\.NETFramework* directory. Local cached copies of .NET Core reference assembly files can be found in the NuGetFallbackFolder directory located under *%ProgramFiles%\\dotnet\\sdk\\* on Windows, */usr/share/dotnet/sdk/* on Linux, or */usr/local/share/dotnet/sdk* on MacOS. Using `DependencyContext` from [Microsoft.Extensions.DependencyModel](https://www.nuget.org/packages/Microsoft.Extensions.DependencyModel/) package can help locate reference assembly files for a particular target platform.
```

Comment:
The paths for the .NET Core reference assemblies are not accurate for .NET Core 3.0.  Also they can be in the NuGet package folder too if the package needs to be downloaded.

Perhaps this doc page shouldn't get into the weeds of exactly what path a reference assembly can be found at.  In normal usage you shouldn't be specifying that path anyway.  For .NET Framework projects, you just specify a simple name reference (ie `<Reference Include="Microsoft.CSharp" />`) and it resolves to the reference assembly.  For .NET Core projects, you have either an implicit or explicit `FrameworkReference` which brings in the reference assemblies, or for versions below .NET Core 3.0 you have an implicit or explicit `PackageReference`.

Reply:
For Visual Studio or MSBuild scenarios, sure you don't need paths. I included that because people sometimes compile by invoking csc from command line or using Roslyn API - in this case they need to specify files directly. Something like this: https://stackoverflow.com/a/32770961/8674428

I'd argue that mentioning these paths at least for .NET Framework makes sence, because they are stable and well-known. But for .NET Core, i agree it's pointless to document them, because these paths are both version dependent and OS dependent, so the list would be too long. Also these reference assemblies are delivered via NuGet packages, so one can simply download package from NuGet gallery and unzip into any directory, or, more correctly, use [PreserveCompilationContext property](https://github.com/dotnet/core/issues/2082#issuecomment-442713181) that lays them out nicely into `publish/refs` subdirectory, so there's no practical need to know these paths. I'll remove them for .NET Core.